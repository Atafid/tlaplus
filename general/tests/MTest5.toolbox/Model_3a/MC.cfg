SPECIFICATION spec_124951961975175000
\* PROPERTY definition
PROPERTY
prop_124951961975176000
\* Generated at Wed Aug 05 17:46:59 PDT 2009