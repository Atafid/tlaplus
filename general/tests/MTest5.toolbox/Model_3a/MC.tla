---- MODULE MC ----
EXTENDS MTest5, TLC

\* Specification formula
spec_124951961975175000 ==
Init /\[][ (x'=x /\ y'=1-y) ]_<<x, y>> 
----
\* PROPERTY definition
prop_124951961975176000 ==
[]<><<y = 1 => y'="a">>_y
----
====
\* Generated at Wed Aug 05 17:46:59 PDT 2009