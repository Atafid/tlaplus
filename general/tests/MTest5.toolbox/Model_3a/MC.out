TLC2 Version 2.02 of 3 August 2009
@!@!@STARTMSG 2187:0 @!@!@
Running in Model-Checking mode.
@!@!@ENDMSG 2187 @!@!@
@!@!@STARTMSG 2220:0 @!@!@
Starting SANY...
@!@!@ENDMSG 2220 @!@!@
Parsing file MC.tla
Parsing file MTest5.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\TLC.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\Naturals.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\Sequences.tla
Semantic processing of module Naturals
Semantic processing of module MTest5
Semantic processing of module Sequences
Semantic processing of module TLC
Semantic processing of module MC
@!@!@STARTMSG 2219:0 @!@!@
SANY finished.
@!@!@ENDMSG 2219 @!@!@
@!@!@STARTMSG 2185:0 @!@!@
Starting...
@!@!@ENDMSG 2185 @!@!@
@!@!@STARTMSG 2212:0 @!@!@
Implied-temporal checking--satisfiability problem has 1 branches.
@!@!@ENDMSG 2212 @!@!@
@!@!@STARTMSG 2189:0 @!@!@
Computing initial states...
@!@!@ENDMSG 2189 @!@!@
@!@!@STARTMSG 2190:0 @!@!@
Finished computing initial states: 1 distinct state generated.
@!@!@ENDMSG 2190 @!@!@
@!@!@STARTMSG 1000:1 @!@!@
Attempted to check equality of integer 0 with non-integer:
"a"
@!@!@ENDMSG 1000 @!@!@
@!@!@STARTMSG 2121:0 @!@!@
The behavior up to this point is:
@!@!@ENDMSG 2121 @!@!@
@!@!@STARTMSG 2217:4 @!@!@
1: <Initial predicate>
/\ x = 0
/\ y = 0

@!@!@ENDMSG 2217 @!@!@
@!@!@STARTMSG 2217:4 @!@!@
2: <Action line 6, col 13 to line 6, col 26 of module MC>
/\ x = 0
/\ y = 1

@!@!@ENDMSG 2217 @!@!@
@!@!@STARTMSG 2103:1 @!@!@
The error occurred when TLC was evaluating the nested
expressions at the following positions:
    The error call stack is empty.

@!@!@ENDMSG 2103 @!@!@
@!@!@STARTMSG 2201:0 @!@!@
The coverage statistics :
@!@!@ENDMSG 2201 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 6, col 13 to line 6, col 16 of module MC: 2
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 6, col 21 to line 6, col 26 of module MC: 2
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2202:0 @!@!@
End of statistics.
@!@!@ENDMSG 2202 @!@!@
@!@!@STARTMSG 2199:0 @!@!@
4 states generated, 2 distinct states found, 0 states left on queue.
@!@!@ENDMSG 2199 @!@!@
@!@!@STARTMSG 2194:0 @!@!@
The depth of the complete state graph search is 2.
@!@!@ENDMSG 2194 @!@!@
@!@!@STARTMSG 2186:0 @!@!@
Finished.
@!@!@ENDMSG 2186 @!@!@
